packages/react-dev-utils/webpackHotDevClient.js

```diff
@@ -25,6 +25,18 @@ var launchEditorEndpoint = require('./launchEditorEndpoint');
 var formatWebpackMessages = require('./formatWebpackMessages');
 var ErrorOverlay = require('react-error-overlay');
 
+ErrorOverlay.listenToOpenInEditor(function OpenInEditor({
```

Comment:
Why a separate function?

Can we just put this back under `ErrorOverlay.startReportingRuntimeErrors` with the prop `launchEditor`?

Reply:
`ErrorOverlay.startReportingRuntimeErrors` and its parameters are specific to runtime errors. But `listenToOpenInEditor` will trigger the listener for both runtime and build errors. So isn't that less intuitive to put `OpenInEditor` under `ErrorOverlay.startReportingRuntimeErrors`?

Reply:
Ah, this makes sense. Thanks!